scripts/eurostat/health_determinants/common/denominator_mcf_generator.py

```diff
@@ -0,0 +1,210 @@
+from dcid_existance import check_dcid_existance
+# pylint: enable = wrong-import-position
+
+_MODULE_DIR = os.path.join(_COMMON_PATH, "..", "bmi")
+
+_INPUT_MCF_FILE_PATH = os.path.join(_MODULE_DIR, "output_files",
+                                    "eurostat_population_bmi.mcf")
+_OUTPUT_MCF_FILE_PATH = os.path.join(_MODULE_DIR, "output_files",
+                                     "eurostat_population_bmi_deno.mcf")
+
+_INCOME_QUINTILE_VALUES = {
+    "IncomeOf0To20Percentile": "[0 20 Percentile]",
+    "IncomeOf20To40Percentile": "[20 40 Percentile]",
+    "IncomeOf40To60Percentile": "[40 60 Percentile]",
+    "IncomeOf60To80Percentile": "[60 80 Percentile]",
+    "IncomeOf80To100Percentile": "[80 100 Percentile]",
+}
+
+_NAME_PROP_REPLACEMENTS = {
+    ", Among": " Among",
+    "Population:,": "Population:",
+    "Percentage,": "Percentage",
+    "Among,": "Among",
+    "ACitizen": "A Citizen",
+    "To": "To ",
+    "Of": "Of ",
+    "To299": "To 299",
+    "To149": "To 149",
+    "Least30": "Least 30",
+    "Normalweight": "Normal Weight"
+}
+
+_ADDITIONAL_NAME_PROP_REPLACEMENTS = {' Among': ''}
+
+
+def _update_name_property(name_prop: str) -> str:
+    """
+    Updates the name property.
+    Args:
+        name_prop (str): name property
+    Returns:
+        str: updated name property
+    """
+
+    name_prop = name_prop.rstrip(', ')
+    name_prop = name_prop.rstrip('with')
+    # Adding spaces before every capital letter,
+    # to make SV look more like a name.
+    name_prop = re.sub(r"(\w)([A-Z])", r"\1 \2", name_prop)
+    name_prop = "name: \"" + name_prop + "\""
+    for key, value in _NAME_PROP_REPLACEMENTS.items():
+        name_prop = name_prop.replace(key, value)
+    if name_prop.strip().endswith('Among"'):
+        for key, value in _ADDITIONAL_NAME_PROP_REPLACEMENTS.items():
+            name_prop = name_prop.replace(key, value)
+    return name_prop
+
+
+# pylint: disable = too-many-return-statements
+# pylint: disable = too-many-branches
+def _generate_pv_node(prop: str) -> str:
+    """
+    Returns the PV for the sv property.
+    Args:
+        prop (str): property
+    Returns:
+        str: property-value
+    """
+
+    if prop in ["Count"]:
+        return "continue"
+    if "PhysicalActivity" in prop:
+        return f"healthBehavior: dcs:{prop}"
+    if "Male" in prop or "Female" in prop:
+        return f"gender: dcs:{prop}"
+    if "Aerobic" in prop or "MuscleStrengthening" in prop \
+        or "Walking" in prop or "Cycling" in prop:
+        return f"exerciseType: dcs:{prop}"
+    if "Education" in prop:
+        return f"educationalAttainment: dcs:{prop.replace('Or', '__')}"
+    if "Percentile" in prop:
+        income_quin = _INCOME_QUINTILE_VALUES[prop]
+        return f"income: {income_quin}"
+    if "Urban" in prop or "Rural" in prop:
+        return f"placeOfResidenceClassification: dcs:{prop}"
+    if "Limitation" in prop:
+        return f"globalActivityLimitationindicator: dcs:{prop}"
```

Comment:
Can you pls update schema for the property to be called: globalActivityLimitationIndicator (caps 'I' for Indicator)?